pkg/workflow/controller/config.go

```diff
@@ -118,6 +120,14 @@ type DindSettings struct {
 	Bip string `json:"bip"`
 }
 
+// WorkersNumber defines workers number for various controller
+type WorkersNumber struct {
```

Comment:
I think it's a little complex.
K8s controller-manager also has various controllers, does it allow to config the concurrency for each controller?

Reply:
I also think this is unnecessary, all controllers can share one value.

Reply:
I separated the config as the workload of every controller is different, and I'd like to make the config more fine-grained.